src/dvclive/live.py

```diff
@@ -379,6 +397,17 @@ def step(self, value: int) -> None:
         self._step = value
         logger.debug(f"Step: {self.step}")
 
+    @property
+    def monitor_cpu(self) -> Optional[MonitorCPU]:
+        return self._monitor_cpu or None
```

Comment:
let's rename it to `_cpu_monitor` (to distinguish from monitor_system that is bool) ? (again, minor, but when I'm reading this code both blend a bit and make me think that there are multiple bool flags - e.g. monitor CPU, monitor GPU in the future that could be enabled / disabled)

Reply:
or may be rename `monitor_system` to make is simpler ... again, not a blocker, just sharing my experience reading this after a break